### PR TITLE
Minor Cursorless docs follow-up tweaks

### LINKED ISSUE #607: Minor Cursorless docs follow-up tweaks
- [ ] Use relative link for `runCommand` [here](https://github.com/cursorless-dev/cursorless-vscode/blob/d90a3698f1fbcc8b8e0c5c14f3851eaf7dfbd05c/docs/contributing/_api-index.md?plain=1#L5).  Might break markdown link checker?
- [x] #609
- [ ] Add link to Cursorless github in top bar

### LINKED ISSUE #609: Have a bit more exposition and point to API doc landing page on contributing landing page.

### PR DESCRIPTION
**Pull request by pokey**
- Fixes #609
- Fixes #607 

## Checklist

- [ ] I have added [tests](https://github.com/cursorless-dev/cursorless-vscode/blob/main/docs/contributing/test-case-recorder.md)

### CODE REVIEW

File: .github/workflows/test-docs.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
         with:
           node-version: 16
           cache: yarn
-      - run: ./scripts/docs-build.sh
+      - run: bash -x scripts/docs-build.sh
```
Comment (pokey) [inline .github/workflows/test-docs.yml:17]: more verbose output

File: scripts/docs-build.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+set -euo pipefail
```
Comment (pokey) [inline scripts/docs-build.sh:1]: Without this line, docs check could fail, but CI would still pass! 😳

File: docs/contributing/_api-index.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
...
-This document is intended to present a high-level overview of the Cursorless architecture, as well as key pointers to locations in the code base to help contributors get up to speed.
+This document is intended to present a high-level overview of the Cursorless
+architecture, as well as key pointers to locations in the code base to help
+contributors get up to speed.
 
-For the time being, we recommend watching the [internals walk-through videos](https://youtube.com/playlist?list=PLkafpFOBVedScHi0dy_80DsHwnZIOSOTy) and having a look through the API docs, starting from the [`runCommand` function](https://www.cursorless.org/docs/contributing/api/classes/core_commandRunner_CommandRunner.CommandRunner#runcommand).
+For the time being, we recommend watching the [internals walk-through
+videos](https://youtube.com/playlist?list=PLkafpFOBVedScHi0dy_80DsHwnZIOSOTy)
+and having a look through the API docs, starting from the [`runCommand`
+function](api/classes/core_commandRunner_CommandRunner.CommandRunner#runcommand).
```
Comment (pokey) [inline docs/contributing/_api-index.md:10]: Relative link now

File: .github/workflows/test-docs.yml [hunk 4]
```diff
@@ -14,9 +14,4 @@ jobs:
         with:
           node-version: 16
           cache: yarn
-      - run: ./scripts/docs-build.sh
-  markdown-link-check:
```
Comment (pokey) [inline .github/workflows/test-docs.yml:18]: Remove markdown link checker.  No longer necessary now that we fail on missing links in docs right?

File: .github/workflows/deploy-docs.yml [hunk 5]
```diff
@@ -11,26 +11,15 @@ jobs:
...
-        run: |
-          echo -n "$GPG_SIGNING_KEY" | base64 --decode | gpg --import
-        env:
-          GPG_SIGNING_KEY: ${{ secrets.CURSORLESS_BOT_GPG_SIGNING_KEY }}
-      - name: git config
-        run: |
-          git config user.name cursorless-bot
-          git config user.email 98099035+cursorless-bot@users.noreply.github.com
-          git config user.signingkey A9387720AFC62221
-          git config commit.gpgsign true
```
Comment (pokey) [inline .github/workflows/deploy-docs.yml:26]: So looking at the commit to `gh-pages`, it appears all this stuff was simply ignored 😅.  I think just using the token and username stuff below should give us the protection we need.

File: docs/contributing/CONTRIBUTING.md [hunk 6]
```diff
@@ -1,16 +1,27 @@
...
 
-### Running tests
+### Running / testing extension locally
 
-```sh
-yarn run test
-```
+In order to test out your local version of the extension or to run unit tests
+locally you need to run the extension in debug mode. To do so you need to run
+the `workbench.action.debug.selectandstart` command and then select either "Run
```
Comment (Will-Sommers) [inline docs/contributing/CONTRIBUTING.md:18]: Super helpful. Realized I was running the tests incorrectly, via the command line or using the test yarn test runner.

Comment (pokey) [reply docs/contributing/CONTRIBUTING.md:18]: Ohhh. Does that fix the issue you were having which required you to switch to insiders?

Comment (Will-Sommers) [reply docs/contributing/CONTRIBUTING.md:18]: Yep!

**Pull request merged**